### ISSUE TITLE
GUI additon

### ISSUE THREAD

**Issue by FormSans** [EngineHub/WorldEdit#1601]
When selecting a space, it is always hard to understand the actual area that has been selected. Sometimes even forgetting where the first selection spot was

I would like at least an option that allows me to actually see what has been selected on screen, able to see the first selected space and the second as well as the space in between.

There use to be other mods that did the GUI but recently I have found that they have been removed.

**Comment by octylFractal** [comment]
We have //drawsel, and I think doing something like WESUI is out of scope for us because of the extra maintenance for cross-platform particle effects. I highly doubt we'll implement any sort of graphical UI in core WorldEdit, but we might create a client mod that works on Forge and Fabric.

WECUI still exists for Fabric at the moment.

**Comment by mk-pmb** [comment]
WE CUI currently (MC 1.20.6, MC 1.21) doesn't work in single player though, because of a double packet type registration issue. ([WE#2554](https://github.com/EngineHub/WorldEdit/pull/2554), [CUI#113](https://github.com/EngineHub/WorldEditCUI/pull/113))

I just discovered that `//drawsel` actually does have an effect in single player – but only if you're in creative mode. I'll investigate whether there is an option to have it in survival single player as well. I heard it has limitations about the size, but for a lot of my uses that's totally ok.

**Comment by me4502** [comment]
drawsel only working in creative mode is a Minecraft limitation, that WorldEdit cannot work around without also becoming a client mod. That's very out of scope for WorldEdit so it's not happening. If you're capable of running client mods then something like WECUI makes more sense. Adding odd workarounds like that doesn't.

**Comment by mk-pmb** [comment]
Yes, fixing WE CUI is my preferred approach. If you check the PRs you'll see that I was already trying to do so, but from the maintainers comments it seems like my approach wouldn't work, and she seems to not have found more time yet to advise on how to proceed.
